src/main/java/seedu/address/logic/commands/ClearCommand.java

```diff
@@ -11,7 +11,7 @@
 public class ClearCommand extends Command {
 
     public static final String COMMAND_WORD = "clear";
-    public static final String MESSAGE_SUCCESS = "Address book has been cleared!";
+    public static final String MESSAGE_SUCCESS = "GME has been cleared!";
```

Comment:
Should the message be "both pool and passenger lists have been cleared"? as suggested in issue #192 
else LGTM